src/react-queries/queries/swapper.ts

```diff
@@ -2,26 +2,46 @@ import { createQueryKeys } from '@lukemorales/query-key-factory'
 import type { ChainId } from '@shapeshiftoss/caip'
 import { type HDWallet, supportsETH } from '@shapeshiftoss/hdwallet-core'
 import type { GetEvmTradeQuoteInput } from '@shapeshiftoss/swapper'
+import type { Asset } from '@shapeshiftoss/types'
 import { arbitrumBridgeApi } from 'lib/swapper/swappers/ArbitrumBridgeSwapper/endpoints'
 import { getTradeQuote } from 'lib/swapper/swappers/ArbitrumBridgeSwapper/getTradeQuote/getTradeQuote'
 
+// Escape-hatch query-key-factory for type safety, and probably ditch it altogether/only keep it for query-keys for now on:
+// https://github.com/lukemorales/query-key-factory/issues/77
+export const arbitrumBridgeTradeQuoteQueryFn = async (
```

Comment:
Move me to `src/lib/swapper/swappers/ArbitrumBridgeSwapper/getTradeQuote/getTradeQuote.ts` as `getTradeQuoteWithWallet` for composibility